src/components/AppProvider/Context.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+import {AppProviderContextType} from './types';
+import createPolarisContext from './utilities/createPolarisContext';
+
+const AppProviderContext = React.createContext<AppProviderContextType>(
```

Comment:
We'll need a naming convention going forward. `AppProviderContext` ideally fits for both the result of `createContext` and the type for the context. @alex-page and I decided `AppProviderContextType` fits but aren't fully sold on it. Open to alternate suggestions 🙏

Reply:
`AppProviderContextType` feels fine by me